platform/p4/docker-sonic-p4.mk

```diff
@@ -1,31 +0,0 @@
-# docker image for p4 sonic docker image
-
-DOCKER_SONIC_P4 = docker-sonic-p4.gz
```

Comment:
>p4 [](http://example.com/codeflow?start=31&length=2)

Is this related to quagga? Why remove?

Reply:
Removing p4 related part is because it uses quagga regardless of the routing stack selected. Updated the PR description to clarify that.